content/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -0,0 +1,295 @@
+documentation for these options lives in the `type Configuration struct`
+[here.](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/plugins.go)
+
+#### Repositories Under Existing Organizations
+
+If you are onboarding a repository in an organization for which plugins are already configured, you will only need to
+enable plugins that you do not already inherit from the organization by adding those plugins under a new
+`plugins.yaml["plugins"]["$org/$repo"]` key. The one plugin you may want to configure at a repository-scoped level is:
+
+|Name|Description|
+|:---|:---|
+|`approve`|enables the `/approve` functionality with `OWNERS` files|
+
+Review the list of plugins enabled for your owning organization and the live [plugin
+catalog](https://prow.ci.openshift.org/plugins) when choosing which plugins you want on your repo.
+
+If your repository does not have `OWNERS files`, you will not be able to opt into the `/approve` process or automatic pull
+request review assignment. `OWNERS` file format and interaction details can be found
+[upstream.](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/approve/approvers/README.md) You will also
+need to add bugzilla component information to your `OWNERS` file. Format can be found
+[here.](https://docs.google.com/document/d/1V_DGuVqbo6CUro0RC86THQWZPrQMwvtDr0YQ0A75QbQ/edit?usp=sharing)
+
+#### Repositories Under New Organizations
+
+If you are onboarding a component not in any organization we already have configured, consider copying the openshift
+organization’s plugin configuration for your organization under a new `plugins.yaml["plugins"]["$org"]` key, or/ setting
+your repo’s configuration without adding any organization-wide configuration by adding the full list of plugins you
+require under a new `plugins.yaml["plugins"]["$org/$repo"]` key.
+
+### Describing Tests
+
+Prow provides the following test trigger types:
+
+|Type Name|Trigger|Target|Purpose|
+|:---|:---|:---|:---|
+|`presubmit`|Push to a PR|A single PR merged into the branch it is targeting|Testing commits within a PR before they are merged|
+|`postsubmit`|Push/merge to a branch|User specified set of branches|Integration tests after a PR is merged|
+|`periodic`|`cron`-like schedule|User-specified set of branches|Scheduled test runs|
+
+Configuration for your repository’s tests live in YAML files in the
+[openshift/release](https://github.com/openshift/release) repository. Jobs are stored in
+many files, sharded by branch and job type, found under: `ci-operator/jobs/$org/$repo/$org-$repo-$branch-$jobtype.yaml`.
+The org and repo name redundancy is because of a requirement that the basename of your YAML file is unique under the
+`ci-operator/jobs` tree. More detailed Prow job configuration documentation lives
+[upstream.](https://github.com/kubernetes/test-infra/tree/master/prow#how-to-add-new-jobs)
+
+#### Bootstrapping Configuration for a new Repository
```

Comment:
Maybe we want this higher up so most people do this first and dig into the Prow specifics later if they need it?